### PR TITLE
Cleanup emotion/babel deps, package.json

### LINKED ISSUE #4192: Remove Babel@6 completely from all dependencies of storybook
Installing storybook apps or addons currently installs babel@6 because of emotion@9. This causes issues for people who are using babel@7 in their apps. But the reverse should be okay.

### Please specify which version of Storybook and optionally any affected addons that you're running

- @storybook/* 4.0.0-alpha.21

### Acceptance criteria

Installing storybook apps or addons should never install babel@6 (only exception can be react native)

### PR DESCRIPTION
**Pull request by pksunkara**
## What I did

- Move to emotion 10 completel
- Cleanup bithound
- Cleanup lerna publishConfig

Fixes #4192

### CODE REVIEW

**Review by shilman — APPROVED** [review]
Awesome!!! Looks like there are some small regressions tho

**Comment by shilman** [comment]
@pksunkara why doesn't the global `publishConfig` work for us?

**Comment by pksunkara** [comment]
It's deprecated in lerna@3 because it was not the correct way to do it. Each scoped package.json is supposed to contain `publishConfig`. https://docs.npmjs.com/files/package.json#publishconfig

**Review by Hypnosphi — APPROVED** [review]

**Comment by Hypnosphi** [comment]
Looks like `jest-emotion` still wants `emotion` package

**Comment by pksunkara** [comment]
Yup, working on it.

**Comment by pksunkara** [comment]
Removed `emotion@9` only from `dependencies` for now so that I can unblock the next alpha release. The `jest-emotion` and `emotion@9` are still being used in `devDependencies` of our tests. Will make a separate PR for it.

File: lib/ui/src/modules/ui/components/search_box.js [hunk 1]
```diff
@@ -102,25 +101,23 @@ class SearchBox extends React.Component {
...
+            background: theme.overlayBackground,
+            zIndex: 1,
+          },
+          content: {
+            fontSize: theme.mainTextColor,
+            fontFamily: theme.mainTextFace,
+            color: theme.mainTextColor,
+          },
+        }}
+        className="tmp" // This exists to make the Modal discard it's default styles for content
```
Comment (Hypnosphi) [inline lib/ui/src/modules/ui/components/search_box.js]: Can we just override the styles that we don't want to have?

Comment (dartharaknus) [reply lib/ui/src/modules/ui/components/search_box.js]: There are too many for content.

Comment (pksunkara) [reply lib/ui/src/modules/ui/components/search_box.js]: Content has a lot of default styles. This was a cleaner way to do it.

**Pull request merged**